package exceptionalhandling.rules;

public class Rule_15 {

    public static void main(String[] args) {

        //You can't define multiple finally block at the below of try block, try and catch block
/*        try {
            System.out.println(10 / 0);
        } catch (ArithmeticException a) {
            a.printStackTrace();
        } finally {
            System.out.println("This is finally block 1");
        } finally {
            System.out.println("This is finally block 2");
        }*/


    }
}
